src/common/arrow_duckdb.cpp

```diff
@@ -0,0 +1,66 @@
+#include "duckdb/common/exception.hpp"
+#include "duckdb/common/assert.hpp"
+#include "duckdb/common/arrow_duckdb.hpp"
+namespace duckdb {
+
+ArrowSchemaDuck::~ArrowSchemaDuck() {
```

Comment:
The Duck suffix there seems superfluous

Reply:
Perhaps `Wrapper` is more apt?

Reply:
Yeah the problem is that ArrowSchema already exists, I can use wrapper instead of duck, I think makes more sense